### ISSUE TITLE
Voice Cloning

### ISSUE THREAD

**Issue by ParthThakkar08** [serp-ai/bark-with-voice-clone#64]
Do we have any options for generating a voice actually matching the input file?

I am trying  to clone the voice for hindi but it changes the voice very much so I wanted to know if we can anyhow create a new model or upgrade the voice we have in model or something? I am also happy to create a new model for it and we can work on this together if you guys don't mind.

Thanks

**Comment by universeroc** [comment]
Yes I have the same problem as the output is changed so much different with the input in tune & style

**Comment by prateek2095** [comment]
Hi, ParthThakkar08 and universeroc. I am also try to train the model for hindi text and voice. i am getting error under train_semantic code in setuping the datasets for the model. If you guy's have something to share please share it. Hoping positive response from your side.

**Comment by devinschumacher** [comment]
@ParthThakkar08 

We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉 Check it out: https://github.com/serpapps/ai-voice-cloner
